### ISSUE TITLE
System.import funcunit somehow stops qunit's beforeEach hook to run

### ISSUE THREAD

**Issue by mickmcgrath13** [bitovi/funcunit#200]
This test fails:
```js
import QUnit from 'steal-qunit';

System.import('funcunit').then(function (F) {
	// F.attach(QUnit);
	let beforeEachHasRun = false;

	QUnit.module('[Functional] System.import funcunit', {
		beforeEach(assert) {
			beforeEachHasRun = true;
		},
		afterEach() {
			beforeEachHasRun = false;
		}
	});

	QUnit.test('before each ran for this test', function (assert) {
		assert.ok(beforeEachHasRun);
	});
	QUnit.test('before each ran for this test, too', function (assert) {
		assert.ok(beforeEachHasRun); //this fails
	});
});
```

..notice that the `F.attach(QUnit)` is commented out...

**Comment by mickmcgrath13** [comment]
Here's a branch with some tests: https://github.com/bitovi/funcunit/tree/200-system-import-funcunit

This passes: [ES6 Import](https://github.com/bitovi/funcunit/blob/200-system-import-funcunit/test/system-import/import-qunit.js) ( [html file](https://github.com/bitovi/funcunit/blob/200-system-import-funcunit/test/system-import/import-qunit.html) )
This Fails: [System.import](https://github.com/bitovi/funcunit/blob/200-system-import-funcunit/test/system-import/system-import-qunit.js) ( [html file](https://github.com/bitovi/funcunit/blob/200-system-import-funcunit/test/system-import/system-import-qunit.html) )

**Comment by matthewp** [comment]
I'm a little confused, it fails because you never attach a test runner, right? You have to attach QUnit or FuncUnit doesn't have a test runner to use.  Maybe I'm misunderstanding the bug though.

**Comment by mickmcgrath13** [comment]
It fails with or without the `F.attach(QUnit)`.  It is commented out to show that isn't anything within the attachment of QUnit to Funcunit that is causing the issue.

**Comment by mickmcgrath13** [comment]
More specifically, we're not actually using anything from FuncUnit in this test.  We're simply wrapping our QUnit test in a `System.import` of funcunit, and that in itself seems to cause QUnit to not behave as expected (namely, the beforeEach runs for the first test but not for the second).

**Comment by matthewp** [comment]
Oh, I see. steal-qunit will automatically run your tests once the `main` has finished loading. Since you are adding tests after the `main` has loaded, steal-qunit has already started QUnit before those tests are defined.

I'm not really sure what a good workaround for that might be.

**Comment by mickmcgrath13** [comment]
there's not a way to tell steal-qunit to wait?  something like
```js
import QUnit from 'steal-qunit';
QUnit.dontStartTestsUntil(System.import("funcunit")).then((F) => { 
  /* ... */ 
});
```

..maybe?

**Comment by matthewp** [comment]
No, maybe steal-qunit could check if `QUnit.config.autostart !== false` before doing its thing, or something like that, in which case you would have to start the tests yourself.

**Comment by matthewp** [comment]
Have you tried using start/stop in your test directly?

```js
import QUnit from 'steal-qunit';

QUnit.stop();

System.import('funcunit').then(function(){
  // add tests here
  
  QUnit.start();
});
```

**Comment by mickmcgrath13** [comment]
I'll ry that and let you know; thanks!

**Comment by mickmcgrath13** [comment]
That gets:
![image](https://cloud.githubusercontent.com/assets/1030325/25091930/2721b63a-2351-11e7-9d50-f14fd3b94ed8.png)

**Comment by matthewp** [comment]
I that case we need some feature added to steal-qunit like [described here](https://github.com/bitovi/funcunit/issues/200#issuecomment-293738060) that will delay starting the tests until you tell it to.
